src/main/scala/com/github/unchama/seichiassist/subsystems/gachapoint/domain/gachapoint/GachaPoint.scala

```diff
@@ -54,15 +66,20 @@ object GachaPoint {
    *   変換にて得られるガチャ券の総数
    */
   case class Usage(remainingGachaPoint: GachaPoint, gachaTicketCount: Int) {
-    require(gachaTicketCount <= GachaPoint.batchSize, "usage must not exceed batch size")
+    require(gachaTicketCount <= GachaPoint.largeBatchSize, "usage must not exceed batch size")
 
     def asTuple: (GachaPoint, Int) = (remainingGachaPoint, gachaTicketCount)
   }
 
   /**
    * ガチャ券へのポイント交換にて一度に得られるガチャ券の上限
    */
-  final val batchSize = 9 * 64
+  final val largeBatchSize = 9 * 64
+  
+  /**
+   * ガチャ券へのポイント交換にて一度に得られるガチャ券の上限
+   */
+  final val smallBatchSize = 64
```

Comment:
`largeBatchSize` の定義に対して書いたレビューの内容を行った後、以下のように変更すると良いと思います。

```suggestion
  /**
   * ガチャ券を64個(= 64 * 1スタック)のバッチでガチャポイントに変換する際のバッチサイズ
   */
  final val smallBatchSize = BatchSize(64)
  
   /**
    * ガチャ券へのポイント交換にて一度に得られるガチャ券の上限
    */
  final val maxBatchSize = largeBatchSize
```